snappy/services.go

```diff
@@ -46,6 +51,14 @@ type interacter interface {
 // wait this time between TERM and KILL
 var killWait = 5 * time.Second
 
+const (
+	// the default target for systemd units that we generate
+	servicesSystemdTarget = "multi-user.target"
+
+	// the default target for systemd units that we generate
+	socketsSystemdTarget = "sockets.target"
+)
+
```

Comment:
Moved back. Let's iterate for a moment, to simplify everything now that there are no circular imports and see what makes sense as the next step in general. I think that, perhaps, much of systemd/ will depend on how we want to shape the REST API for services.